docs: add CONTRIBUTING.md from python-sdk (#10)
open-feature/python-sdk-contrib: CONTRIBUTING.md

Run `make init` to initialize the project's virtual environment and install all dev dependencies.

### Testing

Run tests with `make test`.

We use `pytest` for our unit testing, making use of `parametrized` to inject cases at scale.

### Integration tests

These are planned once the SDK has been stabilized and a Flagd provider implemented. At that point, we will utilize the [gherkin integration tests](https://github.com/open-feature/test-harness/blob/main/features/evaluation.feature) to validate against a live, seeded Flagd instance.

### Packaging

We publish to the PyPI repository, where you can find this package at [openfeature-sdk-contrib](https://pypi.org/project/openfeature-sdk-contrib/).

## Pull Request

All contributions to the OpenFeature project are welcome via GitHub pull requests.

To create a new PR, you will need to first fork the GitHub repository and clone upstream.

```bash
git clone https://github.com/open-feature/python-sdk-contrib.git openfeature-python-sdk-contrib
```

Navigate to the repository folder

```bash

> cd openfeature-python-sdk

Reviewer: ```suggestion
cd openfeature-python-sdk-contrib
```

```

Add your fork as an origin

```bash

> git remote add fork https://github.com/YOUR_GITHUB_USERNAME/python-sdk.git

Reviewer: ```suggestion
git remote add fork https://github.com/YOUR_GITHUB_USERNAME/python-sdk-contrib.git
```

```

Ensure your development environment is all set up by building and testing

```bash
make
```

To start working on a new feature or bugfix, create a new branch and start working on it.

```bash
git checkout -b feat/NAME_OF_FEATURE
# Make your changes
git commit
git push fork feat/NAME_OF_FEATURE
```

> Open a pull request against the main js-sdk repository.

Reviewer: ```suggestion
Open a pull request against the main python-sdk-contrib repository.
```
